addon/models/user.js

```diff
@@ -157,19 +158,21 @@ export default Model.extend({
     }
   ),
 
-  fullName: computed('firstName', 'middleName', 'lastName', function() {
-    const { firstName, middleName, lastName } = this.getProperties('firstName', 'middleName', 'lastName');
+  fullName: computed('firstName', 'middleName', 'lastName', 'displayName', function() {
+    if (this.displayName) {
+      return this.displayName;
+    }
 
-    if (!firstName || !lastName) {
+    if (!this.firstName || !this.lastName) {
       return '';
     }
 
-    const middleInitial = middleName?middleName.charAt(0):false;
+    const middleInitial = this.middleName?this.middleName.charAt(0):false;
 
     if (middleInitial) {
-      return `${firstName} ${middleInitial}. ${lastName}`;
+      return `${this.firstName} ${middleInitial}. ${this.lastName}`;
```

Comment:
should be `${this.middleInitial}` then.

Reply:
This is *exactly* how I messed this up the first time 😛 `middleInitial` isn't a property like the other two, it's defined on 170 as `const middleInitial = this.middleName?this.middleName.charAt(0):false;`.

Reply:
i stand corrected! _approved_